sklearn/metrics/ranking.py

```diff
@@ -172,16 +192,48 @@ def average_precision_score(y_true, y_score, average="macro",
     >>> y_true = np.array([0, 0, 1, 1])
     >>> y_scores = np.array([0.1, 0.4, 0.35, 0.8])
     >>> average_precision_score(y_true, y_scores)  # doctest: +ELLIPSIS
-    0.79...
+    0.83...
+
+    >>> yt = np.array([0, 0, 1, 1])
```

Comment:
you could also just use the y_true and y_score variables defined above, instead of redefining them here?